### PR TITLE
Radial Menus

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Adds radial menus to a LOT of things. Also includes tooltips to make some unclear radial menus clearer.
* Removing and selecting components in hardsuits.
* Selecting tools in polytools and folding knives.
* Choosing mech parts to remove/repair/whatever.
* Removing, toggling, etc. components in cyborgs.
* Selecting between various options in surgery.
* Selecting tools to emulate with the tinker psi power.
* Removing cartridges from a chem dispenser.
* Probably some more I forgot.

### CODE REVIEW

**Comment by BurgerLUA** [comment]
Is this good to merge?

**Comment by MarinaGryphon** [comment]
This is ready to merge after review.

**Comment by MarinaGryphon** [comment]
PR is ready.

**Comment by BurgerLUA** [comment]
This PR is very hold. Are you still working on this?

**Comment by ProbablyCarl** [comment]
Probably going to close this, given it's stale.

**Comment by chaoko99** [comment]
I mean it does work. It's been testmerged for ages. Just hardmerge it.

**Comment by MemedHams** [comment]
I kind of hated them for the first few rounds they were merged, but I got used to them and now that they're gone, I kind of miss 'em.

**Comment by SilvernKell** [comment]
I loathe toolset implants without radials, and it works really well for mech construction, I am indifferent on radials for surgery, but everything else is so much better then those crappy old dialogue box menus.

File: code/game/objects/items/weapons/material/swiss.dm [hunk 1]
```diff
@@ -18,39 +18,44 @@
 	takes_colour = FALSE
 	valid_colors = null
 	max_force = 10
+
+  var/list/tool_icons
```
Comment (MarinaGryphon) [inline code/game/objects/items/weapons/material/swiss.dm:22]:
```suggestion
	var/list/tool_icons
```

**Pull request merged**